tests/test-cases/Group9-VIC-Admin/9-2-VICAdmin-CertAuth.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation  Test 9-2 - VICAdmin CertAuth
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  certs=${true}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+Default Tags
+
+*** Test Cases ***
+Wait
+    Run  sleep 30
+
+Display HTML
+    ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}
+    Log To Console  ${output}
+    Should contain  ${output}  <title>VCH Admin</title>
+
+Get Portlayer Log
+     ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}/logs/port-layer.log
+    Should contain  ${output}  Launching portlayer server
+
+Get VCH-Init Log
+    ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}/logs/init.log
+    Should contain  ${output}  reaping child processes
+
+Get Docker Personality Log
+     ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}/logs/docker-personality.log
+    Should contain  ${output}  docker personality
+
+# Get Container Logs
```

Comment:
That means you're deployed with a DHCP IP, and without a wildcard domain certificate. You need to specify --no-tlsverify in that scenario.

I've got a config for local testing in workstation that just deploys with a static IP in the bridge network. Default config for workstation in my case is that 192.168.x.3 - 192.168.x.127 are not assigned via DHCP.

If you add `--client-network-ip=192.168.218.127 --client-network-gateway=192.168.218.2/24` you should be good.